control/control.go

```diff
@@ -174,6 +175,7 @@ func OptSetConfig(cfg *Config) PluginControlOpt {
 		c.Config = cfg
 		c.pluginManager.SetPluginConfig(cfg.Plugins)
 		c.pluginManager.SetPluginLoadTimeout(c.Config.PluginLoadTimeout)
+		c.pluginRunner.SetPluginLoadTimeout(c.Config.PluginLoadTimeout)
```

Comment:
To be able to access PluginLoadTimeout and set it in runner.go, I would need to add the method SetPluginLoadTimeout to the pluginRunner, similar to pluginManager. Currently, on master, the timeout value is hardcoded in runner.go - https://github.com/intelsdi-x/snap/blob/master/control/runner.go#L183
This PR fixes that by reading it from config, so that it's consistent across LoadPlugin in plugin_manager.go and startPlugin in runner.go.